habitat-baselines/habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -939,11 +943,28 @@ def _eval_checkpoint(
             len(config.habitat_baselines.video_render_views) > 0
             and len(self.config.habitat_baselines.eval.video_option) > 0
         ):
-            with read_write(config):
-                for render_view in config.habitat_baselines.video_render_views:
-                    uuid = config.habitat.simulator[render_view].uuid
-                    config.habitat.gym.obs_keys.append(uuid)
-                    config.habitat_baselines.sensors.append(render_view)
+            for render_view in config.habitat_baselines.video_render_views:
+                uuid: Optional[str] = None
+                for agent_id in config.habitat.simulator.agents:
+                    if (
+                        render_view
+                        in config.habitat.simulator.agents[
+                            agent_id
+                        ].sim_sensors
+                    ):
+                        uuid = (
+                            config.habitat.simulator.agents[agent_id]
+                            .sim_sensors[render_view]
+                            .uuid
+                        )
+                    else:
+                        raise ValueError(
+                            f"Missing render sensor `{render_view}` from agent `{agent_id}`"
+                        )
+                if uuid is not None:
+                    with read_write(config):
+                        config.habitat.gym.obs_keys.append(uuid)
+                        config.habitat.simulator.debug_render = True
```

Comment:
As far as I understand, the `PPOTrainer._eval_checkpoint` expects one agent's checkpoint to be evaluated. If so, then we shouldn't have `for agent in ...` loop and code should look like this:

```suggestion
            from habitat.config.default import get_agent_config
            agent_config = get_agent_config(config.habitat.simulator)
            agent_sensors = agent_config.sim_sensors
            render_view_uuids = [
                agent_sensors[render_view].uuid for render_view in config.habitat_baselines.video_render_views
                if render_view in agent_sensors
            ]
            if not render_view_uuids:
                raise ValueError(
                    f"Missing render sensors `{config.habitat_baselines.video_render_views}` from agent"
                )
            with read_write(config):
                for render_view_uuid in render_view_uuids:
                    if render_view_uuid not in config.habitat.gym.obs_keys:
                        config.habitat.gym.obs_keys.append(render_view_uuid)
                config.habitat.simulator.debug_render = True
```